tests/test_SlotMetaInfoDisplayWidget.py

```diff
@@ -0,0 +1,58 @@
+import pytest
```

Comment:
Our naming convention in volumina seems to be the other way round so, please rename to `slotMetaInfoDisplayWidget_test.py`